src/main.rs

```diff
@@ -139,6 +139,35 @@ enum AutofixMode {
     Yolo,
 }
 
+fn prelude(suggestion: &Replacement) {
+    let snippet = &suggestion.snippet;
+    if snippet.text.1.is_empty() {
+        if suggestion.replacement.ends_with('\n') || suggestion.replacement.starts_with('\n') {
```

Comment:
What are you testing here? What meaning does a replacement starting or ending with a newline have? Can you maybe rewrite this as `let meaningful_name_of_usecase_here = …; if meaningful_name_of_usecase_here {`?

Reply:
öhm... the use case is suggestions which wanna be on their own line... by being inserted after or before another line.

Reply:
Doh! Okay, that is pretty obvious…

-ly to be named `let wants_to_be_on_own_line` :D